examples/Makefile

```diff
@@ -49,6 +49,30 @@ get-python-36:
 get-python-36-verify:
 	kubeless function call get-python-36 |egrep hello.world
 
+get-python-url:
+	kubeless function deploy get-python-url --runtime python2.7 --handler helloget.foo --from-file https://raw.githubusercontent.com/kubeless/kubeless/v1.0.0-alpha.1/examples/python/helloget.py
+
+get-python-url-verify:
+	kubeless function call get-python-url |egrep hello.world
+
+get-python-url-filedep:
+	kubeless function deploy get-python-url-filedep --runtime python2.7 --handler helloget.foo --from-file https://raw.githubusercontent.com/kubeless/kubeless/v1.0.0-alpha.1/examples/python/hellowithdeps.py --dependencies python/requirements.txt
+
+get-python-url-filedep-verify:
+	kubeless function call get-python-url-filedep |egrep Google	
+
+get-python-url-deps:
+	kubeless function deploy get-python-url-deps --runtime python2.7 --handler helloget.foo --from-file https://raw.githubusercontent.com/kubeless/kubeless/v1.0.0-alpha.1/examples/python/hellowithdeps.py --dependencies https://raw.githubusercontent.com/kubeless/kubeless/v1.0.0-alpha.1/examples/python/requirements.txt
+
+get-python-url-deps-verify:
```

Comment:
This is very complete but these kind of tests take a long time to be executed in the CI. Can you merge `get-python-url`, `get-python-url-filedep` and `get-python-url-deps` in just one test? Basically having just `get-python-url-deps` since that one covers the previous two.